copier/template.py

```diff
@@ -300,7 +300,12 @@ def exclude(self) -> Tuple[str, ...]:
 
         See [exclude][].
         """
-        return tuple(self.config_data.get("exclude", DEFAULT_EXCLUDE))
+        return tuple(
+            self.config_data.get(
+                "exclude",
+                [] if Path(self.subdirectory) != Path(".") else DEFAULT_EXCLUDE,
```

Comment:
Would you please invert this condition? I've got [something personal](https://www.recallstack.icu/en/2020/09/01/do-not-use-negative-booleans-please/) against negative conditionals 😂

Reply:
Sure thing, makes sense! :+1: I'll try to remember it. :wink: